libs/wingsdk/src/cloud/topic.ts

```diff
@@ -47,6 +48,16 @@ export class Topic extends Resource {
     props;
     throw new AbstractMemberError();
   }
+
+  /**
+   * Subscribing queue to the topic
+   * @abstract
+   */
+  public subscribeQueue(queue: Queue, props?: TopicOnMessageOptions): void {
```

Comment:
```suggestion
  public subscribeQueue(queue: Queue, props?: TopicSubscribeQueueOptions): void {
```

Reply:
I think the `props` are not necessary in this case, Chris. 

If we were creating the `queue` within this method, it would be necessary, but since we define the `queue` externally and use this method only to bind it to the `topic`, I believe we can remove the `props`.

Reply:
Makes sense 👍